### ISSUE TITLE
Feature request: android app global server url setting

### ISSUE THREAD

**Issue by poblabs** [binwiederhier/ntfy#127]
In the android app right now, the server url setting seems to be specific to UnifiedPush. It would be great if there was another global setting for the self-hosted server URL. This way you wouldn't have to enter it in under "use another server" every time you sub to a new topic. Just a thought since the "use another server" is pre-filled once you use it once.

**Comment by binwiederhier** [comment]
So I love this idea and I did want to express that I very much agree that this should exist. Thank you for reporting it.

There's a slight "problem" though with the UX here:

The Google Play flavor only supports Firebase for ntfy.sh (a plaform limitation); plus, we have the "instant delivery" checkbox, which is only relevant for ntfy.sh, since all other hosts implicitly have instant delivery enabled.

So long story short, I don't quite know how to reconcile this in a single easy to understand dialog.

I bet I can come up with something, but it's not simple I think.

**Comment by binwiederhier** [comment]
Let's see how hard/easy this is. I'll do this next.

**Comment by binwiederhier** [comment]
This is done and will be in the next app release
